src/core/util/sys_vars.cpp

```diff
@@ -1257,14 +1257,14 @@ void mce_sys_var::get_env_params()
     }
 
     if ((env_ptr = getenv(SYS_VAR_TX_BUF_SIZE)) != NULL) {
-        tx_buf_size = (uint32_t)atoi(env_ptr);
+        tx_buf_size = (uint32_t)option_size::from_str(env_ptr);
```

Comment:
why not use to_str in printing for these as well?

Reply:
We can, willl update.

Reply:
note that rx_buf_size is not displayed in the header. Updated only tx_buf_size and zc_tx_size